script/prep-travis-release.sh

```diff
@@ -92,7 +99,7 @@ BASENAME="rack"
 RACKBUILD="${BASENAME}${SUFFIX}"
 
 COMMIT=$(git rev-parse --verify HEAD)
-sed -i "s/var Commit =.*/var Commit = \"$COMMIT\"/" util/util.go
+sed -i "s/var Commit =.*/var Commit = \"$COMMIT\"/" util/commit.go
```

Comment:
Are we sure? I thought that was always coming from the code. Is the concern that the code doesn't line up with what travis says?